ToDoApp/containers/home/Home.js

```diff
@@ -1 +1,90 @@
-//File
+import React, { Component } from 'react';
+import {TouchableOpacity, Text, View, ScrollView, StyleSheet } from 'react-native';
+import {createStackNavigator, createAppContainer} from 'react-navigation';
+import Item from '../../components/item/Item';
+import ButtonCustom from '../../components/button/ButtonCustom';
+import Detail from '../detail/Detail';
+import NewTask from '../newTask/NewTask';
+
+var data = [
+    {
+        'first-description': 'Description',
+        'second-description': 'Second Description',
+        'status': false
+    },
+    {
+        'first-description': 'Description',
+        'second-description': 'Second Description',
+        'status': false
+    },
+    {
+        'first-description': 'Description',
+        'second-description': 'Second Description',
+        'status': false
+    },
+    {
+        'first-description': 'Description',
+        'second-description': 'Second Description',
+        'status': true
+    }, 
+    {
+        'first-description': 'Description',
+        'second-description': 'Second Description',
+        'status': false
+    }
+];
+
+class Home extends Component {
+  static navigationOptions = ({navigation}) => ({
+    title: 'Home',
+    headerLeft: (<View />),
+    headerRight: (
+        <View style={{width:25,height:20}}>
```

Comment:
same about styles

Reply:
I have just changed that, in the next push it will be added.